test/commandlineflags_gtest.cc

```diff
@@ -34,14 +34,30 @@ TEST(BoolFromEnv, False) {
   ASSERT_EQ(setenv("BENCHMARK_IN_ENV", "0", 1), 0);
   EXPECT_EQ(BoolFromEnv("in_env", true), false);
   unsetenv("BENCHMARK_IN_ENV");
+
+  ASSERT_EQ(setenv("BENCHMARK_IN_ENV", "false", 1), 0);
+  EXPECT_EQ(BoolFromEnv("in_env", true), false);
+  unsetenv("BENCHMARK_IN_ENV");
+
+  ASSERT_EQ(setenv("BENCHMARK_IN_ENV", "F", 1), 0);
+  EXPECT_EQ(BoolFromEnv("in_env", true), false);
+  unsetenv("BENCHMARK_IN_ENV");
+
+  ASSERT_EQ(setenv("BENCHMARK_IN_ENV", "no", 1), 0);
+  EXPECT_EQ(BoolFromEnv("in_env", true), false);
+  unsetenv("BENCHMARK_IN_ENV");
+
+  ASSERT_EQ(setenv("BENCHMARK_IN_ENV", "N", 1), 0);
+  EXPECT_EQ(BoolFromEnv("in_env", true), false);
+  unsetenv("BENCHMARK_IN_ENV");
 }
 
 TEST(BoolFromEnv, True) {
   ASSERT_EQ(setenv("BENCHMARK_IN_ENV", "1", 1), 0);
   EXPECT_EQ(BoolFromEnv("in_env", false), true);
   unsetenv("BENCHMARK_IN_ENV");
 
-  ASSERT_EQ(setenv("BENCHMARK_IN_ENV", "foo", 1), 0);
+  ASSERT_EQ(setenv("BENCHMARK_IN_ENV", "bar", 1), 0);
   EXPECT_EQ(BoolFromEnv("in_env", false), true);
```

Comment:
Fun: `"foo"` starts with an `f` which evaluates to `false` for `IsTruthyFlagValue()`... Would it be an acceptable breakage to change `IsTruthyFlagValue()`'s logic for something more strict? Any other suggestion welcome of course.

Reply:
i think as long as it supports:

T, t, yes, true, True, TRUE as true and
F, f, no, false, False, FALSE as false

we can accept it.